src/util/arrayToSentence.js

```diff
@@ -0,0 +1,5 @@
+import arrayToSentence from 'array-to-sentence';
+
+export function arrayToSentenceWithOr(array) {
+  return arrayToSentence(array, {lastSeparator: ' or '});
```

Comment:
Can we pull `' or '` out of the translation file, and call this something like `localizedArrayToSentence`?